apps/examples/testcase/tc_main.c

```diff
@@ -69,6 +69,9 @@
 #if defined(CONFIG_EXAMPLES_TESTCASE_TASK_MANAGER_UTC)
 #define TC_TASK_MANAGER_STACK  2048
 #endif
+#if defined(CONFIG_EXAMPLES_TESTCASE_TASK_MANAGER_ITC)
```

Comment:
I think you'd better merge this defined condition with the above
like below:
#if defined(CONFIG_EXAMPLES_TESTCASE_TASK_MANAGER_UTC) || defined(CONFIG_EXAMPLES_TESTCASE_TASK_MANAGER_ITC)

Reply:
done